exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,19 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+  
+function createGreeting() {
```

Comment:
there should be 2 functions, one that spells the mentor's name in upper case and one that creates a shouty greeting.